cookbooks/ambari-server/recipes/ambari.rb

```diff
@@ -0,0 +1,125 @@
+# Cookbook Name:: ambari-server
+# Description : Recipe to install Ambari-server and TEZ libraries on a given host and creates a TEZ View Instance. 
+
+
+ambari_repo_url = node[:ambari][:repo][:url]
+hdp_repo_url = node[:ambari][:hdp][:repo][:url]
+repo_dir = node[:ambari][:repo][:dir]
+ambari_server_ip = node[:ambari][:server]
+instance_name = node[:ambari][:view][:instance_name]
+AMBARIUSER = node[:ambari][:database][:username]
+AMBARISERVERFQDN = node[:ambari][:server]
+AMBARIDATABASE = node[:ambari][:database][:name]
+yarn_timelineserver_url="http://<%= node['ambari']['yarn']['timelineserver']['host']%>:8188"
+yarn_resourcemanager_url="http://<%= node['ambari']['yarn']['resourcemgr']['host']%>:8088"
+ambari_url= "http://<%= node['ambari']['server']%>:8080/api/v1/views/TEZ/versions/0.5.2.2.2.2.0-151/instances"
+make_config('ambari-database-password', secure_password(64))
+node.override[:ambari][:database][:password] = get_config('ambari-database-password')
+AMBARIPASSWORD = node[:ambari][:database][:password]
+
+#Variables used in creating TEZ view
+
+yarn_timelineserver_url = node[:ambari][:yarn][:timelineserver][:url]
+yarn_resourcemanager_url = node[:ambari][:yarn][:resourcemanager][:url]
+instance_name = node[:ambari][:view][:instance_name]
+ambari_uri = node[:ambari][:server][:url]
+
+
+
+remote_file "/tmp/ambari.deb" do
+  source "#{get_binary_server_url}/ambari-server-withoutdependencies.deb"
+  owner "root"
+  group "root"
+  mode "755"
+mode "0644"
+end
+
+bash "Installing Ambari server" do
```

Comment:
Why can't we use package resource? Our bootstrap host also serves a debian repository and package can be installed from there instead of downloading it locally.